tests/benchmarks/graph500_scale21-2hops.yml

```diff
@@ -0,0 +1,22 @@
+name: "graph500_scale21-2hops"
+description: "graph500 scale 21 dataset contains 2396019 nodes and and 67108864 edges"
+remote:
+  - setup: redisgraph-r5
+  - type: oss-standalone
+dbconfig:
+  - dataset: "https://s3.amazonaws.com/benchmarks.redislabs/redisgraph/datasets/graph500_scale21/graph500_scale21_v2.2.16_dump.rdb"
+  - dataset_load_timeout_secs: 1200
```

Comment:
Would have been nice if we could reuse the same server to perform all 3 benchmarks, this can save us the 2 extra RDB load